.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+        run: pnpm run typecheck
+
+      - name: "🏗️ Build applications"
+        run: pnpm run build
+        env:
+          NEXT_PUBLIC_APP_NAME: CricketScore
+          NEXT_PUBLIC_APP_DESCRIPTION: Your ultimate source for real-time cricket scores and updates
+          NEXT_PUBLIC_API_BASE_URL: https://cricketapi-r9zv.onrender.com
+          NEXT_PUBLIC_ENABLE_REAL_TIME: true
+          NEXT_PUBLIC_ENABLE_NOTIFICATIONS: false
+          NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY: ${{ secrets.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY }}
+          CLERK_SECRET_KEY: ${{ secrets.CLERK_SECRET_KEY }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)